ts/components/staking/CurrentEpochOverview.tsx

```diff
@@ -0,0 +1,81 @@
+import { BigNumber } from '@0x/utils';
+import { differenceInCalendarDays } from 'date-fns';
+import * as React from 'react';
+import styled from 'styled-components';
+import { colors } from 'ts/style/colors';
```

Comment:
super tiny nit: this import (ln 5) should probably be with the other local imports